frontend/src/TooljetDatabase/Forms/RowForm.jsx

```diff
@@ -259,22 +325,37 @@ const RowForm = ({ onCreate, onClose, referencedColumnDetails, setReferencedColu
                 placeholder={
                   isSerialDataTypeColumn ? 'Auto-generated' : inputValues[index]?.value !== null && 'Enter a value'
                 }
-                className={
-                  isSerialDataTypeColumn && !darkMode
-                    ? 'primary-idKey-light'
-                    : isSerialDataTypeColumn && darkMode
-                    ? 'primary-idKey-dark'
-                    : !darkMode
-                    ? 'form-control'
-                    : 'form-control dark-form-row'
-                }
+                className={`
+                  ${
+                    isSerialDataTypeColumn && !darkMode
```

Comment:
Use `cx` from `classnames` package, when revising existing className. It is a tech debt, we can fix them slowly when we change the class names